### DISCUSSION TITLE
Self-Hosting the UI

### DISCUSSION THREAD

**Discussion by ngregrichardson** [Lodestone-Team/lodestone#253] [Q&A] [answered]
First of all, I love this project. The idea is excellent, and the UI is *chefs kiss*. However, I'm confused about self-hosting it. I am currently running Lodestone in a Docker container, and the only way I can see to connect to the UI is through the [website](https://lodestone.cc). Am I missing the ability to self-host that UI as well? Is that served from the Docker container, should I deploy the website separately, or is this not supported yet?

**Comment by CheatCod** [comment, marked as answer]
Thanks for the kind words! The dashboard is a standard react project so you can just clone https://github.com/Lodestone-Team/lodestone and do `npm i` and `npm build`

**Comment by trbntwo** [comment]
When using the Container Image for the webapp, which ports need to be opened/mapped?

EDIT: found it myself, seems to be port 80

**Reply by viedok** [reply]
Which image did you use to be able to run the webapp in docker ? I assume it isn't going to work with the "lodestone_core" one..

**Reply by trbntwo** [reply]
This one:
https://github.com/Lodestone-Team/lodestone/pkgs/container/lodestone_dashboard
